x/oracle/keeper/tally.go

```diff
@@ -18,18 +19,20 @@ func (k Keeper) Tally(ctx sdk.Context, voteIterator sdk.Iterator, voteType types
 	k.setOracleValidatorInfosInTally(ctx, tally)
 
 	for ; voteIterator.Valid(); voteIterator.Next() {
+		vote := reflect.New(reflect.ValueOf(voteType).Elem().Type()).Interface().(types.Vote)
```

Comment:
I'm not sure if this is the best way to copy the interface. 🤔

Reply:
I understood what you tried to do. This will work, but I think that reflection codes are hard to read. It can cause mistakes in the future. 
So, I'd suggest a simpler way: https://github.com/medibloc/panacea-core/commit/e1e6328cd87221f83ba60a4d014252409b0683fe.

Reply:
Thank you!
I fixed it. (https://github.com/medibloc/panacea-core/pull/403/commits/9cbdc8929f9a81333c6cae92cf6e071a749476b2)